### ISSUE TITLE
Automatic Source PWA.Metadata.Categories from Metadata.LANG.Categories.List

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#90]
# Description
```
Please provide a short description of what feature you're looking forward to
enhance below. Please include the story behind your idea as well to give a
better visualization of your idea.
```
Right now, `PWA.Metadata.Categories` and `Metadata.LANG.Categories.List` are duplicates of one another.  Since PWA settings are dynamically support, we can remove this duplicates once and for all.

# Expected Behavior
```
Please specify the expected behavior of your requested enhancement. Some great
and helpful pointers are your expected interface (e.g. command patterns,
simple sketches of the user interface, etc).
```
When `PWA.Metadata.Categories` are left empty, it shall automatically source from `Metadata.LANG.Categories.List` for all languages.

# Current Behavior
```
Please specify the current behavior (e.g. workaround, blockage, etc).
```
Duplicates are a must.

# Attachment
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Implemented in f350c9ab354350917f7cad9fdde348c43072451e

**Comment by hollowaykeanho** [comment]
Released in version `v.1.2.0`.
